playbooks/ORCHESTRATION.md

```diff
@@ -0,0 +1,219 @@
+# Orchestration
+
+This playbook describes how to orchestrate a root signing event.
+
+## Pre-work
+
+1. Check the [calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing events. 
+
+2. Make updates to the targets and delegation configuration files, see [configuration](#targets-and-delegation-configuration).
+
+3. Double-check the configured [role expirations](https://github.com/sigstore/root-signing/blob/e3f1fe5e487984f525afc81ac77fa5ce39737d0f/cmd/tuf/app/init.go#L28).
+
+4. Set environment variables, including previous repository and online signer references (details [here](#key-configuration)). 
+
+| Variable      | Description | Example |
+| ----------- | ----------- | ----------- | 
+| GITHUB_USER      | The GitHub user, used to create PRs and commit messages       | asraa       |
+| BRANCH   | The working branch, in case of staging script or configuration changes.        | main        |
```

Comment:
Does BRANCH and LOCAL have to be set always, I thought those were just for testing?

Reply:
"If enabled" &  "in case of staging script or configuration changes", i tried to make that clear? added "testing"/optional

Reply:
Perhaps just start the description with "(Optional, used when...)"?

Reply:
Yep! marked as optional in the update